src/utils/getTeams.ts

```diff
@@ -1,31 +1,31 @@
 import * as Sentry from '@sentry/node';
 
-import { GH_ORGS, PRODUCT_OWNERS_YML } from '@/config';
+import { GH_ORGS, PRODUCT_OWNERS_INFO } from '@/config';
 
 export function getTeams(
   repo: string,
-  productArea: string,
-  org: string
+  org: string,
+  productArea: string | undefined
```

Comment:
nit: for consistency, probably `productArea?: string`?

Reply:
done